### PR TITLE
Tweaks for step

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: demo/mobile/package.json [hunk 1]
```diff
@@ -1,101 +1,96 @@
...
-    "load-grunt-tasks": "3.4.1",
-    "time-grunt": "1.3.0",
-    "uglifyify": "~3.0.4"
-  },
-  "cordova": {
-    "platforms": [
-      "android",
-      "ios",
-      "browser"
+    "name": "@raincatcher/demo-mobile",
```
Comment (paolobueno) [inline demo/mobile/package.json]: Indentation changed here to 4 spaces for some reason.

**Review by paolobueno — APPROVED** [review]
Thanks for drying out the extra plugins. Just needs reindenting the package.json file.

File: steps/examples/step-gallery/lib/angular/urlProvider.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+module.exports = function getUrl(FhLibrary) {
+  return new Promise(function(resolve, reject) {
+    // Get server url
+    $fh.on('fhinit', function(error) {
```
Comment (paolobueno) [inline steps/examples/step-gallery/lib/angular/urlProvider.js]: Maybe you mean to use `FhLibrary` here?

File: steps/examples/step-gallery/lib/angular/urlProvider.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      if (error) {
+        return reject(error);
+      }
+      resolve(decodeURIComponent($fh.getCloudURL()));
```
Comment (JameelB) [inline steps/examples/step-gallery/lib/angular/urlProvider.js]: `FHLibrary` here too

**Review by paolobueno — APPROVED** [review]

File: demo/portal/src/app/main.js [hunk 4]
```diff
@@ -48,7 +48,7 @@ angular.module('app', [
   vehicleInspectionStep.ngModule(),
   accidentStep.ngModule(),
   signatureStep.ngModule(),
-  cameraStep.ngModule()
+  galleryStep.ngModule($fh)
```
Comment (JameelB) [inline demo/portal/src/app/main.js:51]: $fh here is not defined

**Pull request merged**